imod_coupler/drivers/dfm_metamod/dfm_metamod.py

```diff
@@ -314,6 +330,51 @@ def exchange_flux_riv_active_mf62dfm(self) -> None:
             )[:]
         )
 
+    def exchange_ponding_msw2dflow1d(self) -> None:
+        # MetaSWAP delivers m3/dtsw -> m3/s
+        msw_ponding_volume = self.msw.get_surfacewater_ponding_allocation()
+        msw_ponding_flux_sec = msw_ponding_volume / days_to_seconds(self.dtsw)
+        dflow1d_flux_receive = self.dfm.get_1d_river_fluxes()
+        if dflow1d_flux_receive is None:
+            raise ValueError("dflow 1d river flux not found")
+        dflow1d_flux_receive = dflow1d_flux_receive + (
+            self.mask_msw_dflow1d["msw-ponding2dflow1d_flux"][:]
+            * dflow1d_flux_receive[:]
+            + self.map_msw_dflow1d["msw-ponding2dflow1d_flux"].dot(
+                msw_ponding_flux_sec
+            )[:]
+        )
+
+    def exchange_sprinkling_msw2dflow1d(self) -> None:
+        # MetaSWAP delivers m3/dtsw -> m3/s
+        msw_sprinkling_demand = self.msw.get_surfacewater_sprinking_demand()
+        msw_sprinkling_demand_sec = msw_sprinkling_demand / days_to_seconds(self.dtsw)
+        dflow1d_flux_receive = self.dfm.get_1d_river_fluxes()
+        if dflow1d_flux_receive is None:
+            raise ValueError("dflow 1d river flux not found")
+        dflow1d_flux_receive = dflow1d_flux_receive + (
+            self.mask_msw_dflow1d["msw-sprinkling2dflow1d_flux"][:]
+            * dflow1d_flux_receive[:]
+            + self.map_msw_dflow1d["msw-sprinkling2dflow1d_flux"].dot(
+                msw_sprinkling_demand_sec
+            )[:]
+        )
+
+    def exchange_sprinkling_dflow1d2msw(
```

Comment:
yes, they add to the dflow 1d node flux since they supply (ponding) and withdraw (sprinkling (negative)) from the surface water system. In the current situation they are adding to the water balance